torch_xla/csrc/tensor_methods.cpp

```diff
@@ -2639,6 +2641,21 @@ XLATensorPtr view(const XLATensorPtr& input,
   return input->CreateViewTensor(std::move(view_info));
```

Comment:
@wonjoolee95 @alanwaketan I vaguely remember we reuse a lot of old View code even during functionization pass, do we have a plan to remove them completely? For example ideally we shouldn't call `CreateViewTensor` and drop `viewinfo` class.

Reply:
Yea, there is a task to replace those hacks with more formal lowering: https://docs.google.com/document/d/1d1u8oLrL7iZz55NbCfQSYzwLP4M60kb7igvdBVE_X7s/edit?resourcekey=0-W3QvBNHNPUeX84-TkRJjlA#bookmark=id.tux3bndmql98